server/src/main/java/org/apache/iotdb/db/engine/storagegroup/DataRegion.java

```diff
@@ -3541,7 +3542,9 @@ public IDTable getIdTable() {
   }
 
   public IWALNode getWALNode() {
-    if (!config.isClusterMode()) {
+    if (!config
+        .getDataRegionConsensusProtocolClass()
+        .equals(ConsensusFactory.MultiLeaderConsensus)) {
       throw new UnsupportedOperationException();
     }
```

Comment:
Is there any problem if started as a standalone IoTDB?

Reply:
No, only multi-leader consensus will invoke this method.